docs/topics/feed-exports.rst

```diff
@@ -185,6 +185,27 @@ passed through the following settings:
  * :setting:`AWS_ACCESS_KEY_ID`
  * :setting:`AWS_SECRET_ACCESS_KEY`
 
+.. _topics-feed-storage-gcs:
+
+Google Cloud Storage (GCS)
+--------------------------
+
+The feeds are stored on `Google Cloud Storage`_.
+
+ * URI scheme: ``gcs``
+ * Example URIs:
+
+   * ``gcs://mybucket/path/to/export.csv``
```

Comment:
gsutil uses `gs://` as an URI prefix. See here: https://cloud.google.com/storage/docs/gsutil.

Reply:
That's a good one.
For media pipelines, it is also `gs` https://docs.scrapy.org/en/latest/topics/media-pipeline.html#id1
I'll change it :smile:

Reply:
@victor-torres , I updated the scheme!